src/core/Form/Select/SingleSelect/SingleSelect.md

```diff
@@ -1,445 +1,568 @@
+The `<SingleSelect>` component is used to select one item from a list of options. The list can also be filtered by typing.
+
+If you need to select multiple options, use the <a href="#/Components/MultiSelect">MultiSelect</a> component instead.
+
```

Comment:
Should this also mention Dropdown ?  "if you don't need filtering use dropdown" or something?

Reply:
This is now fixed in the latest commit